src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+    protected LocalDate dueDate;
+    public Deadlines(String name, String dueDate) {
+        super(name, "D");
+        this.dueDate = LocalDate.parse(dueDate);
+    }
+
+    public Deadlines(String name, String dueDate, boolean isDone) {
+        super(name, "D", isDone);
+        this.dueDate = LocalDate.parse(dueDate);
+    }
+
+//    protected static LocalDateTime convertToDateFormat(String date) {

```

Comment:
It will be better to delete unwanted comment codes according to the coding standards. When needed there will be old version to reference from.

Reply:
Deleted. Thank you!